File: compatLib/compatLibVQ/src/main/java/app/lawnchair/compatlib/ten/QuickstepCompatFactoryVQ.java
package app.lawnchair.compatlib.ten;

import android.window.RemoteTransition;
import androidx.annotation.NonNull;
import androidx.annotation.RequiresApi;
import app.lawnchair.compatlib.ActivityManagerCompat;
import app.lawnchair.compatlib.ActivityOptionsCompat;
import app.lawnchair.compatlib.QuickstepCompatFactory;
import app.lawnchair.compatlib.RemoteTransitionCompat;

@RequiresApi(29)
public class QuickstepCompatFactoryVQ implements QuickstepCompatFactory {
    protected final String TAG = getClass().getCanonicalName();

    @NonNull
    @Override
    public ActivityManagerCompat getActivityManagerCompat() {
        return new ActivityManagerCompatVQ();
    }

    @NonNull
    @Override
    public ActivityOptionsCompat getActivityOptionsCompat() {
        return new ActivityOptionsCompatVQ();
    }

    @NonNull
    @Override
    public RemoteTransitionCompat getRemoteTransitionCompat() {
        return RemoteTransition::new;
    }
}

File: compatLib/compatLibVQ/src/app/lawnchair/compatlib/ten/QuickstepCompatFactoryVQ.kt
package app.lawnchair.compatlib.ten;

import app.lawnchair.compatlib.ActivityManagerCompat
import app.lawnchair.compatlib.ActivityOptionsCompat
import app.lawnchair.compatlib.eleven.QuickstepCompatFactoryVR

class QuickstepCompatFactoryVQ : QuickstepCompatFactoryVR() {

    override fun getActivityManagerCompat(): ActivityManagerCompat {
        return ActivityManagerCompatVQ()
    }

    override fun getActivityOptionsCompat(): ActivityOptionsCompat {
        return ActivityOptionsCompatVQ()
    }
}
